mangaki/mangaki/utils/svd.py

```diff
@@ -52,7 +52,7 @@ def make_matrix(self, X, y):
             matrix[user][work] = rating
         means = np.zeros((self.nb_users,))
         for i in range(self.nb_users):
-            means[i] = np.sum(matrix[i]) / np.sum(matrix[i] != 0)
+            means[i] = np.nanmean(matrix[i])
```

Comment:
To me: `np.sum(matrix[i]) / np.count_nonzero(matrix[i])` is fine as a replacement (assuming `np.count_nonzero(matrix[i]) != 0`), am I right?